wiki/Beatmapping/Beats_per_minute/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - BPM
+  - Tempo
```

Comment:
```suggestion
  - tempo
```
doesnt matter lol but might as well be consistent since everything else is lowercase already.